src/AutoRest.CSharp/LowLevel/Output/OperationConfidenceChecker.cs

```diff
@@ -82,7 +82,7 @@ private static ConvenienceMethodConfidenceLevel WalkType(InputType type, TypeFac
                 InputModelType modelType => WalkModelType(modelType, typeFactory, visitedModels),
                 InputListType listType => WalkListType(listType, typeFactory, visitedModels),
                 InputDictionaryType dictType => WalkDictionaryType(dictType, typeFactory, visitedModels),
-                InputLiteralType literalType => WalkLiteralType(literalType, typeFactory),
+                InputEnumType enumType => WalkEnumType(enumType, typeFactory),
                 InputUnionType unionType => WalkUnionType(unionType),
                 _ => ConvenienceMethodConfidenceLevel.Confident
```

Comment:
would this change make the case of `InputLiteralType` previously to get out from the literal type branch, but now get out from the default branch?

Reply:
`InputLiteralType` is replaced at the earlier stage, so this branch goes to enum.